feeds.conf.default

```diff
@@ -0,0 +1,11 @@
+src-git packages https://github.com/openwrt/packages.git
+src-git luci https://github.com/openwrt/luci.git
+src-git routing https://github.com/openwrt-routing/packages.git
+src-git telephony https://github.com/openwrt/telephony.git
+src-git management https://github.com/openwrt-management/packages.git
+src-git targets https://github.com/openwrt/targets.git
+src-git pkg https://github.com/CreatorDev/openwrt-feeds.git
```

Comment:
pkg->ci40
we have used ci40 in vanila feeds.conf.default, lets stick to same nomenclature